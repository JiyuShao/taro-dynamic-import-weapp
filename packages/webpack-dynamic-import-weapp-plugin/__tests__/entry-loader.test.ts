import EntryLoader from '../src/entry-loader';

describe('DynamicImportWeappPlugin entry-loader', () => {
  it('"entry-loader" should return prefix code', () => {
    expect(EntryLoader('script end')).toMatchSnapshot();
  });
});
