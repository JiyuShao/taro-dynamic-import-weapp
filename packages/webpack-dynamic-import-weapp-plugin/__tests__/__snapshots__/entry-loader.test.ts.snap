// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DynamicImportWeappPlugin entry-loader "entry-loader" should return prefix code 1`] = `
"
import * as eval5 from 'eval5';
import * as TaroComponents from '@tarojs/components';

wx['eval5'] = eval5;
Object.keys(TaroComponents);

script end
"
`;
